script/Ciel/Day 13/13_01_02_CIEL13_7C.txt

```diff
@@ -939,52 +1092,61 @@ The girl's single thrust could be seen as the roar of her soul.
 [sha:9ca18d42de70a0895ed6fecaf1114332f51034b7]{
 -- Page 346, Offset 36283.
 -- それを甘んじて受けたいと想う<情|じょう>ごと、彼女は敵を粉砕した。
-Along with her desire to accept those feelings wholeheartedly, she lays waste to her foe.
+// Assuming this is like 甘んじて受け入れる?
+// https://www.weblio.jp/content/%E7%94%98%E3%82%93%E3%81%98%E3%81%A6%E5%8F%97%E3%81%91%E5%85%A5%E3%82%8C%E3%82%8B
+// As in, Ciel doesn't like that she has to do this?
+Wishing she could resign herself to what she is about to do,
+ the Executor lays waste to her foe.
 }
 [sha:3f44784efc10c21664904a63dbe398b383fe2705]{
 -- Page 347, Offset 36284.
 -- 「あ―――、あ、ああ…………あ」
-"Ah―――ah, aah......ah."
+"%{i}Ah%{/i}――%{i}ah%{/i}, %{i}aah%{/i}......%{i}ah%{/i}."
 }
 [sha:f9f62dd998c3fe2c68b945f7baa823a2caa014ff]{
 -- Page 347, Offset 36285.
 -- 杭打ち機は少女の真芯を貫いた。
-The pile-driver pierces through the girl's core.
+The pile bunker punches straight through the girl's chest.
 }
 [sha:061481b3f974c3d30eed7da95c7067391b88542a]{
 -- Page 347, Offset 36286.
 -- カラン、と鉄の槍が地面に落ちる。
-With a clank, her iron spear falls to the ground.
+Her iron spear clanks as it hits the ground.
```

Comment:
I like the previous.

Reply:
For some reason the 'with an $onomatope' formulation really bothers me. I think it just crops up a lot if not managed.

Reply:
I can see why, but the sentence has more impact in the double clause structure. There's something odd about describing it like this. I have hard time exactly vocalizing why though.